ports/geckolib/glue.rs

```diff
@@ -2351,7 +2352,8 @@ pub extern "C" fn Servo_DeclarationBlock_SetFontFamily(declarations:
     let string = unsafe { (*value).to_string() };
     let mut input = ParserInput::new(&string);
     let mut parser = Parser::new(&mut input);
-    if let Ok(family) = FontFamily::parse(&mut parser) {
+    let result = FontFamily::parse(&mut parser);
```

Comment:
something something borrow checker something temporaries created in the last statement of a block live as long as the block something. No idea why that didn’t occur before, though.